### PR TITLE
Update kubeadm version

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: genref/output/md/kubeadm-config.v1beta3.md [hunk 1]
```diff
@@ -0,0 +1,1459 @@
...
+	apiServer:
+	  extraArgs:
+	    authorization-mode: "Node,RBAC"
+	  extraVolumes:
+	  - name: "some-volume"
+	    hostPath: "/etc/some-path"
+	    mountPath: "/etc/some-pod-path"
+	    readOnly: false
+	    pathType: File
+	  certSANs:
```
Comment (kbhawkey) [inline genref/output/md/kubeadm-config.v1beta3.md:213]: Do you want to remove these tabs?

File: genref/config.yaml [hunk 2]
```diff
@@ -179,3 +179,9 @@ apis:
     title: kubeadm Configuration (v1beta2)
     package: github.com/tengqm/kubeconfig
     path: config/kubeadm/v1beta2
+
+  # This is for testing
```
Comment (kbhawkey) [inline genref/config.yaml:183]: Is this used for testing?

File: genref/output/md/kubeadm-config.v1beta3.md [hunk 3]
```diff
@@ -0,0 +1,1459 @@
...
+	 advertiseAddress: "10.100.0.1"
+	 bindPort: 6443
+certificateKey: "e6a2eb8581237ab72a4f494f30285ec12a9694d750b9785706a83bfcbbbd2204"
+---
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+etcd:
+  # one of local or external
+  local:
+	   imageRepository: "k8s.gcr.io"
```
Comment (kbhawkey) [inline genref/output/md/kubeadm-config.v1beta3.md:182]: tabs?

File: genref/output/md/kubeadm-config.v1beta3.md (hunk 1 above)
Comment (tengqm) [reply genref/output/md/kubeadm-config.v1beta3.md:213]: oh ??? These are from the source code. Will investigate and remove them.

File: genref/config.yaml (hunk 2 above)
Comment (tengqm) [reply genref/config.yaml:183]: Yes. The package is deeply embedded in k/k repo. To generate reference docs for kubeadm, we have to clone the relevant files somewhere (currently under github.com/tengqm/kubeconfig as you can see from line 180).

File: genref/output/md/kubeadm-config.v1beta2.md [hunk 4]
```diff
@@ -955,7 +949,7 @@ DNS defines the DNS add-on that should be used in the cluster
 - [DNS](#kubeadm-k8s-io-v1beta2-DNS)
 
 
-DNSAddOnType defines string identifying DNS add-on types.
+DNSAddOnType defines string identifying DNS add-on types
```
Comment (kbhawkey) [inline genref/output/md/kubeadm-config.v1beta2.md:952]: @tengqm , Is this change coming from the upstream source code?

Comment (kbhawkey) [reply genref/output/md/kubeadm-config.v1beta2.md:952]: It is a minor issue. The formatting is not consistent.

Comment (tengqm) [reply genref/output/md/kubeadm-config.v1beta2.md:952]: Yes. It is from source code change.

File: genref/output/md/kubeadm-config.v1beta2.md [hunk 5]
```diff
@@ -156,90 +153,90 @@ configuration types to be used during a `kubeadm init` run.
...
-  criSocket: "/var/run/dockershim.sock"
-  taints:
-  - key: "kubeadmNode"
-    value: "master"
-    effect: "NoSchedule"
-  kubeletExtraArgs:
-    cgroup-driver: "cgroupfs"
-  ignorePreflightErrors:
-  - IsPrivilegedUser
+	 name: "ec2-10-100-0-1"
```
Comment (kbhawkey) [inline genref/output/md/kubeadm-config.v1beta2.md:167]: nit: tabs or spaces?

**Comment by kbhawkey** [comment]
There are a bunch of minor formatting issues from the source.
/lgtm

File: genref/output/md/kubeadm-config.v1beta2.md (hunk 5 above)
Comment (tengqm) [reply genref/output/md/kubeadm-config.v1beta2.md:167]: Will revisit this. These tabs come from the source code too.

**Pull request merged**